index.html

```diff
@@ -395,6 +395,28 @@ <h2>Convert a <var>name</var> to a <var>timestamp</var></h2>
         The <a data-cite="NAVIGATION-TIMING#performancetiming">PerformanceTiming</a> interface was defined in [[NAVIGATION-TIMING]] and is now considered obsolete. The use of names from the <a data-cite="NAVIGATION-TIMING#performancetiming">PerformanceTiming</a> interface is supported to remain backwards compatible, but there are no plans to extend this functionality to names in the <a data-cite="NAVIGATION-TIMING-2#dom-performancenavigationtiming">PerformanceNavigationTiming</a> interface defined in [[NAVIGATION-TIMING-2]] (or other interfaces) in the future.
       </p>
     </section>
+  </section><section id="recommended-mark-names" class="informative">
```

Comment:
I'm not sure this can be non-normative since it is adding a bunch of definitions

Reply:
Yeah, we can remove the class=informative bit